packages/office-ui-fabric-react/src/components/ResizeGroup/ResizeGroup.test.tsx

```diff
@@ -43,22 +86,121 @@ describe('ResizeGroup', () => {
     const onReduceData = sinon.spy();
     const onRenderData = sinon.spy();
 
-    const div = document.createElement('div');
-
     let wrapper = mount(<ResizeGroup
       data={ { a: 1 } }
       onReduceData={ onReduceData }
       onRenderData={ onRenderData }
     />);
 
-    wrapper = mount(<ResizeGroup
-      data={ { a: 2 } }
-      onReduceData={ onReduceData }
-      onRenderData={ onRenderData }
-    />);
+    wrapper.setProps({
```

Comment:
It was already happening after the first render, removed them from the setProps call for clarity